### ISSUE TITLE
AttributeError: 'iDevice' object has no attribute 'secret64'

### ISSUE THREAD

**Issue by MrJarnould** [thehappydinoa/iOSRestrictionBruteForce#11]
I am now getting this error.  Any help would be appreciated. 

```
 python3 ioscrack.py -m 

 iOSRestrictionBruteForce
 Written by thehappydinoa 

Please navigate to `~/Library/Application\ Support/MobileSync/Backup/`

Copy the contents of `~/Library/Application\ Support/MobileSync/Backup/` to `/Users/jacob/PycharmProjects/iOSRestrictionBruteForce/iOSRestrictionBruteForce/backup`

I'll wait

Press [return] to continue 
ioscrack.py:70: DeprecationWarning: The readPlist function is deprecated, use load() instead
  self.info = plistlib.readPlist(INFOPATH)
Traceback (most recent call last):
  File "ioscrack.py", line 224, in <module>
    main()
  File "ioscrack.py", line 211, in main
    mojave_help()
  File "ioscrack.py", line 156, in mojave_help
    crack_hashes(find_hashes(path=path))
  File "ioscrack.py", line 129, in crack_hashes
    pin = device.crack()
  File "ioscrack.py", line 111, in crack
    self.pin = crack(self.secret64, self.salt64)
AttributeError: 'iDevice' object has no attribute 'secret64'
```

**Comment by LopesX** [comment]
Some issue

**Comment by MrJarnould** [comment]
> Some issue

From the research I've done, it looks like the restrictions file is not located in `"/398bc9c2aeeab4cb0c12ada0f52eea12cf14f40b"` in iOS 12.

**Comment by thehappydinoa** [comment]
Oh damn really?

**Comment by MrJarnould** [comment]
see [this](https://github.com/gwatts/pinfinder/issues/33)

**Comment by thehappydinoa** [comment]
Hopefully, this was fixed in d8d6647 🤞
